server.js

```diff
@@ -8,7 +8,13 @@ app.get("/", (req, res) => {
 })
 
 app.get("/static/bundle.js", (req, res) => {
-  res.sendFile("bundle.js", { root: __dirname })
+  var browserify = require("browserify");
```

Comment:
Eventually we probably want to cache this, instead of re-bundling on every request, but it's OK for now, I think.